app/js/actions/actionTypes.js

```diff
@@ -1,2 +1,14 @@
-/*eslint-disable*/
+import types from 'redux-types';
+
+const BASIC_TYPES = [
+  'SUCCEEDED',
+  'FAILED',
+  'REQUESTED',
+];
+
 export const FETCH_LAB_ORDERS = 'FETCH_LAB_ORDERS';
+
+export default {
+  SET_PATIENT: types('set_patient', BASIC_TYPES),
+  SET_PATIENT_NOTE: types('set_patient_note', BASIC_TYPES),
```

Comment:
Let's just remove "SET_PATIENT_NOTE" for now... it was something added to the header, but we never use it...